resources/ts/Components/Integrations/Detail/BillingInfo.tsx

```diff
@@ -75,17 +75,19 @@ export const BillingInfo = ({
           <Heading level={4} className="font-semibold">
             {t("details.billing_info.to_pay")}
           </Heading>
-          <div className="w-full block relative md:min-w-[40rem]">
-            {t(`pricing_plan.basic.price`, {
-              price: formatPricing({
-                currency: subscription.currency,
-                price: Math.max(
-                  subscription.fee / 100 - (coupon?.reduction ?? 0),
-                  0
-                ),
-              }),
-            })}
-          </div>
+          {coupon && (
```

Comment:
Wouldn't that condition hide the price even in case of no coupon?

Reply:
Yes this isn't needed, nice catch